### PR TITLE
Refs #22310 - provide a script to set origin for existing reports

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by bastilian** [comment]
:+1: LGTM

File: lib/tasks/rescan_reports_origins.rake [hunk 1]
```diff
@@ -0,0 +1,23 @@
+desc 'Rescan existing reports without origin and tries to find correct origin'
+task :rescan_reports_origins do
+  puts "Scanning #{ConfigReport.count} reports, this can take a long time, it's safe to interrupt and rerun later..."
+  User.as_anonymous_admin do
+    ConfigReport.where(:origin => nil).includes(:logs => [:source, :message]).find_in_batches(batch_size: 100) do |group|
+      group.each do |report|
+        Foreman::Plugin.report_scanner_registry.report_scanners.each do |scanner|
+          logs = report.logs.map do |log|
```
Comment (bastilian) [inline lib/tasks/rescan_reports_origins.rake:8]: On second look, should this maybe be limited to only the last log?

Comment (bastilian) [reply lib/tasks/rescan_reports_origins.rake:8]: Nah, we need all. :D `PuppetReportScanner` only needs the first, but the AnsibleReportScanner required more to scan.

Comment (ares) [reply lib/tasks/rescan_reports_origins.rake:8]: :dart: :-)

File: lib/tasks/fix_db_params.rake [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class ConfigReport < Report
+  def metrics
+    self[:metrics]
+  end
+end
+
+class ForemanOpenscap::ArfReport < Report
```
Comment (timogoebel) [inline lib/tasks/fix_db_params.rake]: Does this blow if OpenScap is not installed?

Comment (ares) [reply lib/tasks/fix_db_params.rake]: sorry, fat fingers push, this does not belong to this PR

**Review by dLobatog — APPROVED** [review]
Thanks @ares

**Comment by ares** [comment]
Thanks for merging! @xprazak2 do we already have some place to track what needs to be in 1.18 release notes? This should be probably covered.

**Pull request merged**